companies/123greetings.json

```diff
@@ -1,22 +1,23 @@
 {
-    "slug": "123greetings",
+    "slug": "123greeting2s",
     "relevant-countries": [
-        "all"
+        "all",
+        "dong"
```

Comment:
🚫 **[recordlint]** <[pass-schema](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record did not pass schema validation: “The value at /relevant-countries/1 must be one of: "all", "at", "be", "bg", "hr", "cy", "cz", "dk", "ee", "fi", "fr", "de", "gr", "hu", "ie", "it", "lv", "lt", "lu", "mt", "nl", "pl", "pt", "ro", "sk", "si", "es", "se", "gb", "no", "is", "li", or "ch".”

Raw error:
```json
{
    "path": "$.relevant-countries[1]",
    "pointer": "/relevant-countries/1",
    "message": "The value at /relevant-countries/1 must be one of: \"all\", \"at\", \"be\", \"bg\", \"hr\", \"cy\", \"cz\", \"dk\", \"ee\", \"fi\", \"fr\", \"de\", \"gr\", \"hu\", \"ie\", \"it\", \"lv\", \"lt\", \"lu\", \"mt\", \"nl\", \"pl\", \"pt\", \"ro\", \"sk\", \"si\", \"es\", \"se\", \"gb\", \"no\", \"is\", \"li\", or \"ch\".",
    "data": "dong"
}
```